index.js

```diff
@@ -1,55 +1,94 @@
 const WebSocket = require('ws');
 const request = require('request');
 const getParser = require('@streammedev/parse-message');
+const program = require('commander');
 
-const botKey = 'ADD THE BOT KEY';
-const botSecret = 'ADD THE BOT SECRET';
+program
+	.version('0.0.1')
+	.option('-k, --key <key>', 'bot key')
+	.option('-s, --secret <secret>', 'bot secret')
+	.option('-i, --id <id>', 'your public id')
+	.parse(process.argv);
+	
+var botKey = program.key;
+var botSecret = program.secret;
+var publicId = program.id;
```

Comment:
Moving the interface as suggested in another comment will fix this, but its generally bad practice to leave these things in a global scope
- it stands a chance of leaking
- more importantly, it loosens all of your function/class definitions and makes your interfaces weaker because they require random environmental information instead of this-> attributes or function parameters